app/components/App/App.js

```diff
@@ -18,7 +17,9 @@ class App extends Component {
             <Route exact path='/' component={Play} />
             <Route exact path='/instructions' component={Instructions}/>
             <Route path='/learn' component={Learn} />
-            <p className='copyright-notice'>&#169; 2017 Leta Keane</p>
+            <footer>
+              <p className='copyright-notice'>&#169; 2017 Leta Keane</p>            
+            </footer>
```

Comment:
since you're not managing state etc, does this need to be a `class` ?
you could do something like 
const App = () => {
  <Router>
    ...some routes here
  </Router>
}

Also, why not extract the `footer` into it's own component and then render that wherever it needs to be viewed?